include/swift/Basic/OptionSet.h

```diff
@@ -151,7 +152,7 @@ class OptionSet {
                               Flags>::value,
                 "operator| should produce an OptionSet");
 };
-
+} // end namespace optionset
```

Comment:
Does `using namespace` has any effect in Swift? If not, how about
```suggestion
} // end namespace optionset
using namespace optionset;
```
And that's it (with some doc-comment for this weird hack).

Reply:
`using namespace` is ignored by Swift when it imports C++ headers.

Reply:
Oh, nice idea. I hadn’t thought of that. I tried `using OptionSet = optionset::OptionSet`, which didn’t work but this is much cleaner (and doesn’t require any downstream changes).